server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java

```diff
@@ -231,6 +232,10 @@ protected DescPriorityMergeReader getDescPriorityMergeReader() {
     return new DescPriorityMergeReader();
   }
 
+  public static Filter defaultTimeFilter(boolean ascending) {
+    return ascending ? TimeFilter.gtEq(Long.MIN_VALUE) : TimeFilter.ltEq(Long.MAX_VALUE);
+  }
+
```

Comment:
You can move the method into `TimeFilter` class and add java doc for this method to indicate that it will return a timefilter thta will always satisfy any time.

Reply:
Thanks for the comment, it has been fixed.